README.md

```diff
@@ -19,19 +30,76 @@ pip install --upgrade pip setuptools wheel
 pip install -e .
 ```
 
+Or with a Conda virtual environment:
+
+```
+. ~/miniconda3/etc/profile.d/conda.sh
+conda create -n deepdrivemd python=3.7.9
+conda activate deepdrivemd
+pip install --upgrade pip setuptools wheel
+conda install scipy (this step is needed if a failure of installing scipy is observed)
+pip install -e .
+```
+
 Then, install pre-commit hooks: this will auto-format and auto-lint _on commit_ to enforce consistent code style:
 
 ```
 pre-commit install
 pre-commit autoupdate
 ```
 
-In some places, DeepDriveMD relies on external libraries to configure MD simulations and import specific ML models.
+2. Install OpenMM:
+
+- by source code (for Linux ppc64le, e.g., Summit)
+https://gist.github.com/lee212/4bbfe520c8003fbb91929731b8ea8a1e
+
+- by conda (for Linux x86\_64, e.g., PSC Bridges)
+```
+module load anaconda3
+module load cuda/9.2
+source /opt/packages/anaconda/anaconda3-5.2.0/etc/profile.d/conda.sh
+conda install -c omnia/label/cuda92 openmm
+```
+
+3. In some places, DeepDriveMD relies on external libraries to configure MD simulations and import specific ML models.
 
 For MD, install the `mdtools` package found here: https://github.com/braceal/MD-tools
 
+```
+git clone https://github.com/braceal/MD-tools.git
+pip install .
+```
+
 For ML (specifically the AAE model), install the `molecules` package found here: https://github.com/braceal/molecules/tree/main
 
+```
+git clone https://github.com/braceal/molecules.git
+pip install .
+```
+
+#### Stage: machine_learning
+
+1. Install the `deepdrivemd` virtual environment as above (`deepdrivemd` is needed in all the virtual environments since each task uses the DDMD_API to communicate with the outputs of other tasks).
+
+2. Install the `keras-CVAE` model with `rapidsai DBSCAN` package found here: https://www.ibm.com/docs/en/wmlce/1.6.2?topic=installing-mldl-frameworks
+
+```
+conda config --prepend channels https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/
+conda install powerai-rapids
+```
+
+3. Install packages `scikit-learn` and `h5py` version 2.10.0:
+
+```
+conda install scikit-learn h5py=2.10.0
```

Comment:
does h5py have to be pinned to 2.10?

Reply:
It may have to be to use the virtual H5 API. I have to investigate it more thoroughly but I think for now we should pin to 2.10.0 since that is the version we have tested up to this point and I think I recall having issues with the virtual h5py API with later versions.

There are work arounds to avoid the virtual h5py API, it's something we can update in the future.